example/lib/main.dart

```diff
@@ -20,32 +20,19 @@ Future<void> _handleNotification(RemoteMessage message) async {
       .handleNotification(message, false, rootNavigatorKey);
 }
 
-final GlobalKey<ScaffoldMessengerState> rootScaffoldMessengerKey =
-    GlobalKey<ScaffoldMessengerState>();
-
-final GlobalKey<NavigatorState> rootNavigatorKey = GlobalKey<NavigatorState>();
-
 void main() async {
-  // Loading .env file
-  await dotenv.load(fileName: ".env");
-
   WidgetsFlutterBinding.ensureInitialized();
-
   // Loading .env file
   await dotenv.load(fileName: ".env");
-  await LMFeedCore.instance.initialize(
-      domain: "feedsx://www.feedsx.com/",
-      );
-
   SystemChrome.setPreferredOrientations([
     DeviceOrientation.portraitUp,
     DeviceOrientation.portraitDown,
     DeviceOrientation.landscapeLeft,
     DeviceOrientation.landscapeRight,
   ]);
-
+   await LMFeedCore.instance.initialize();
```

Comment:
We can remove this line from here right? Every user will be shown the cred screen and upon submitting the details we will eventually call this method

Reply:
if we call it `onSubmit` of cred_screen submit button, then DIService will throw an exception like, `notification services are already initialize`, when a user goto the cred screen by poping up from feed_screen and again submitting the other user details, to previent this exception, i have initalized LMFeedCore for only once inside `main()` and just setting the theme and other configuration and widgets on click of submit button, you can check `_setUpFeed()` method on `cred_screen.dart` for more clearity